tests/tests/test_setup_efm.py

```diff
@@ -61,4 +62,34 @@ def test_setup_efm_pg_read_all_settings():
         efm_index = [idx for idx, role in enumerate(result) if 'efm' in role][0]
 
     assert 'pg_read_all_settings' in result[efm_index], \
-        "EFM role is not a pg_read_all_settings"
\ No newline at end of file
+        "EFM role is not a pg_read_all_settings"
+
+def test_setup_efm_redhat():
+    if not get_os().startswith('centos') and not get_os().startswith('rocky'):
+        pytest.skip()
+
+    host = get_primary()
+    packages = [
+        'java-1.8.0-openjdk',
+        'mailx',
+        'edb-efm44'
+    ]
+
+    for package in packages:
+        assert host.package(package).is_installed, \
+            "Package %s not installed" % packages
+
+def test_setup_efm_debian():
+    if not (get_os().startswith('debian') or get_os().startswith('ubuntu')):
+        pytest.skip()
+
+    host = get_primary()
+    packages = [
+        'openjdk-8-jdk',
+        'bsd-mailx',
+        'edb-efm44'
+    ]
+
+    for package in packages:
+        assert host.package(package).is_installed, \
+            "Package %s not installed" % packages
```

Comment:
Including changes not related to the main subject of the PR is not a good practice. Please open a new PR if you want to apply changes on EFM tests.